packages/cli/__tests__/zosfiles/__system__/download/uss/cli.files.download.uss.system.test.ts

```diff
@@ -183,7 +181,7 @@ describe("Download USS File", () => {
 
         it("should download an uss file with response timeout", () => {
             const shellScript = path.join(__dirname, "__scripts__", "command", "command_download_uss_file.sh");
-            const response = runCliScript(shellScript, TEST_ENVIRONMENT, [ussname.substr(1, ussname.length), "--responseTimeout 5"]);
+            const response = runCliScript(shellScript, TEST_ENVIRONMENT, [ussname, "--responseTimeout 5"]);
```

Comment:
Do we know why we were removing the first character on these tests to begin with?

Reply:
Not a clue.